### PR TITLE
Discuss stable points in the continuum

### LINKED ISSUE #9: equilibrium points
The equilibria thing that ekr raised on the list has hit a nerve.  The document should address that and make the point that the protocol lifecycle has different points in the strictness-postel continuum.  Related to this, the stasis point for different implementations might depend on their relative market position.

### PR DESCRIPTION
**Pull request by DavidSchinazi**
Fixes #9

### CODE REVIEW

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -188,13 +188,24 @@ as quirks in early implementations can affect all subsequent deployments.
...
-impossible - to rectify the situation.
+From observing widely deployed protocols, it appears there are two stable points
+on the spectrum between being strict versus permissive in the presence of
+protocol errors:
 
-Interoperability requirements for protocol implementations are set by other
-deployments.  Specifications and - where they exist - conformance test suites
-might guide the initial development of implementations, but implementations
-ultimately need to interoperate with deployed implementations.
+* If there is a majority of strict implementations, newer implementations will
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:195]:
```suggestion
* If implementations predominantly enforce strict compliance with specifications, newer implementations will
```

File: draft-iab-protocol-maintenance.md [hunk 2]
```diff
@@ -188,13 +188,24 @@ as quirks in early implementations can affect all subsequent deployments.
...
+  experience failures if they do not conform to the protocol and will fix them
+  before deploying, adding a new strict implementation to the majority.
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:197]:
```suggestion
  experience failures if they do not comply with protocol requirements. Newer implementations need to
  fix compliance issues in order to be successfully deployed. This ensures
  that most deployments are compliant.
```

I think that compliance > conformance here.  Both arms describe a process that guarantees conformance with group norms.  It's just that one cleaves to a specification that is agreed and written down, the other evolves organically.

File: draft-iab-protocol-maintenance.md [hunk 3]
```diff
@@ -188,13 +188,24 @@ as quirks in early implementations can affect all subsequent deployments.
...
+
+* Conversely, if there is a majority of non-conformant implementations, newer
+  implementations will accumulate protocol non-conformant deviations to the
+  protocol in order to interoperate before they are deployed, adding a new
+  non-conformant implementation to the majority.
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:202]:
```suggestion
* Conversely, if non-compliance is tolerated by existing implementations, non-compliant
  implementations can be deployed successfully.  Newer implementations then
  have strong incentive to tolerate any existing non-compliance in order to be
  successfully deployed.  This ensures that most deployments are tolerant of the same
  non-compliant behavior.
```

**Review by martinthomson — COMMENTED** [review]
I like the general shape of this, but as you can see, I think that it could be rephrased a little.

**Comment by DavidSchinazi** [comment]
I agree with your edits so I landed them

**Pull request merged**